### ISSUE TITLE
Fastlane screenshots failed with flutter project.

### ISSUE THREAD

**Issue by zinwalin** [flutter/flutter#18466]
```
------------------ Environment ----------
fastlane 2.97.0
Flutter 0.4.5-pre.84 • channel beta • https://github.com/flutter/flutter.git
Framework • revision 70772ea5a0 (3 weeks ago) • 2018-05-24 13:57:52 -0700
Engine • revision 86d96fd0ce
Tools • Dart 2.0.0-dev.55.0.flutter-97b6c2e09d
-----------------------------------
```

It happened on iOS platform, not tried on Android yet.

First, I create a flutter app using following command:

```sh
flutter create --org abc.mobile -i swift -a kotlin OneApp
```

Then, I initialise fastlane for iOS project,  
Fastfile:
```
default_platform(:ios)

platform :ios do
  desc "Push a new release build to the App Store"
  lane :release do
    increment_build_number(xcodeproj: "Runner.xcodeproj")
    build_app(workspace: "Runner.xcworkspace", scheme: "Runner")
    screenshots
    upload_to_app_store
  end

  desc "Generate new localized screenshots"
  lane :screenshots do
    capture_screenshots(workspace: "Runner.xcworkspace", scheme: "Runner")
  end
end
```

Snapfile:
```
# Uncomment the lines below you want to change by removing the # in the beginning

# A list of devices you want to take the screenshots from
 devices([
#   "iPhone 8",
#   "iPhone 8 Plus",
#   "iPhone SE",
   "iPhone X",
#   "iPad Pro (12.9-inch)",
#   "iPad Pro (9.7-inch)",
#   "Apple TV 1080p"
 ])

 languages([
   "en-US"
 ])

# The name of the scheme which contains the UI Tests
 scheme("Runner")

# Where should the resulting screenshots be stored?
 output_directory("./screenshots")

# remove the '#' to clear all previously generated screenshots before creating new ones
 clear_previous_screenshots(true)

# Arguments to pass to the app on launch. See https://docs.fastlane.tools/actions/snapshot/#launch-arguments
# launch_arguments(["-favColor red"])

# For more information about all available options run
# fastlane action snapshot
```

then I created UI Testing target for iOS project and import SnapshotHelper.swift which generated by fastlane.  fastlane screenshots will use UI Tests to take screenshot, but failed with following errors:
`"x86_64" is not an allowed value for option "ios-arch".`

when I run `flutter run -d all ` with a running simulator first, then fastlane screenshots will success.

**Comment by thosakwe** [comment]
The solution is here: https://github.com/flutter/flutter/issues/17735#issuecomment-400837152

**Comment by zoechi** [comment]
@zinwalin does the suggestion solve your problem?

**Comment by AppleEducate** [comment]
when running the test from Xcode I get this error
`Library not loaded: @rpath/Flutter.framework/Flutter`
And the Test Fails.

and when running` fastlane snapshot`

`[13:37:24]: ▸ ❌  fatal error: file '/Users/Shared/CrossMedia/Projects/MyUnifyMobile/ios/Runner/GeneratedPluginRegistrant.h' has been modified since the precompiled header '/var/folders/8v/39qx36ss40s_c18typ5773m40000gn/T/snapshot_derived20180712-16281-how46e/Build/Intermediates.noindex/PrecompiledHeaders/Runner-Bridging-Header-swift_1K1BX6OPSA6RK-clang_34PNLM54WL6LG.pch' was built`

**Comment by AppleEducate** [comment]
@zinwalin The fix from #17735 fixed xcode being able to create the test, but once it is created you cannot rerun the test from xcode.

**Comment by gliese667c** [comment]
> when running the test from Xcode I get this error
> 
> `Library not loaded: @rpath/Flutter.framework/Flutter`
> 
> And the Test Fails.
> 
> 
> 
> and when running` fastlane snapshot`
> 
> 
> 
> `[13:37:24]: ▸ ❌  fatal error: file '/Users/Shared/CrossMedia/Projects/MyUnifyMobile/ios/Runner/GeneratedPluginRegistrant.h' has been modified since the precompiled header '/var/folders/8v/39qx36ss40s_c18typ5773m40000gn/T/snapshot_derived20180712-16281-how46e/Build/Intermediates.noindex/PrecompiledHeaders/Runner-Bridging-Header-swift_1K1BX6OPSA6RK-clang_34PNLM54WL6LG.pch' was built`

Hello 
@rodydavis 
How did you fix it? I've been stuck on this two days
